### PR TITLE
Fix mapbox wrapper viewstate support

### PR DESCRIPTION
**Pull request by Pessimistress**
- Fix crash if initial view state is provided as object
- Default value fallbacks in viewState properties

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,7 +23,6 @@
   ],
   "scripts": {
     "start": "(cd examples/main && (path-exists node_modules || npm i) && npm run start-local)",
-    "clean": "rm -fr dist dist-es6 && mkdir -p dist/es5 dist/esm dist/es6",
```
Comment (ibgreen) [inline package.json]: Probably for another PR, but don't we want a `scripts/clean.sh?` or `scripts/build.sh clean`?

File: src/mapbox/mapbox.js [hunk 2]
```diff
@@ -286,7 +286,15 @@ export default class Mapbox {
...
   _getViewState(props) {
-    return props.viewState || props;
+    const {
+      longitude,
+      latitude,
+      zoom,
+      pitch = 0,
+      bearing = 0,
+      altitude = 1.5
+    } = props.viewState || props;
```
Comment (ibgreen) [inline src/mapbox/mapbox.js]: Should we drop support for `props` and just look at `props.viewState`?
Or just issue a deprecation warning if `props.viewState` is not set?

Comment (Pessimistress) [reply src/mapbox/mapbox.js]: There is no plan to deprecate flat props in react-map-gl. Doesn't make sense to switch to `viewState` without multi-viewport.

File: package.json (hunk 1 above)
Comment (Pessimistress) [reply package.json]: No reason to keep it separate. We published the last alpha with redundant files because `clean` was not called with `build`.

Comment (ibgreen) [reply package.json]: No reason to keep it separate. 

> Not for build purposes. My thinking was that a typical usage for a `clean` script is to let the user to clean out built and installed artifacts from repo / the source tree, as they can take a lot of space. Not critical.

**Pull request merged**